### DISCUSSION TITLE
Disk space available on VMs

### DISCUSSION THREAD

**Discussion by alelievr** [codemagic-ci-cd/codemagic-docs#1460] [Q&A] [answered]
Hello,

I'm trying to Build a big Unity project (almost 60GB once cloned) and I get a `not enough disk space` error while downloading the Unity editor version we want to use.
Right now I'm testing things with the free plan, is there more space available on the Teams plan?

I found in the documentation pages that specify the disk space available depending on the config we choose, for [macOS](https://docs.codemagic.io/specs/versions-macos/) it's supposed to be `87GB` of free space which should be enough to build our project. Is this a limitation of the free tier?

Here's my build ID: 6407c58306904f039780447d. The out-of-space error is in the `Retrieve Used Unity Version And Install` section.

**Comment by icarusdust** [comment, marked as answer]
Hi @alelievr, try removing some softwares from the machine by adding the following script at the top of your scripts section:

```
sudo rm -rf /Applications/Xcode-13.3.app
sudo rm -rf /Applications/Xcode-13.4.app
sudo rm -rf /Applications/Xcode-14.0.app 
sudo rm -rf /Applications/Xcode-14.3.app 
```

See if it helps.
